packages/suite-build/configs/base.webpack.config.ts

```diff
@@ -163,16 +163,31 @@ const config: webpack.Configuration = {
     },
     plugins: [
         new webpack.ProgressPlugin(),
-        new webpack.DefinePlugin({
-            'process.browser': true,
-            'process.env.SUITE_TYPE': JSON.stringify(project),
-            'process.env.VERSION': JSON.stringify(suiteVersion),
-            'process.env.COMMITHASH': JSON.stringify(gitRevision),
-            'process.env.ASSET_PREFIX': JSON.stringify(assetPrefix),
-            'process.env.JWS_PUBLIC_KEY': JSON.stringify(jwsPublicKey),
-            'process.env.CODESIGN_BUILD': isCodesignBuild,
-            'process.env.SENTRY_RELEASE': JSON.stringify(sentryRelease),
-        }),
+        new webpack.DefinePlugin(
+            isDev
+                ? {
+                      'process.browser': true,
+                      'process.env.SUITE_TYPE': JSON.stringify(project),
+                      'process.env.VERSION': JSON.stringify(suiteVersion),
+                      'process.env.COMMITHASH': JSON.stringify(gitRevision),
+                      'process.env.ASSET_PREFIX': JSON.stringify(assetPrefix),
+                      'process.env.JWS_PUBLIC_KEY': JSON.stringify(jwsPublicKey),
+                      'process.env.CODESIGN_BUILD': isCodesignBuild,
+                      'process.env.SENTRY_RELEASE': JSON.stringify(sentryRelease),
+                  }
+                : {
+                      'process.browser': true,
+                      'process.env.SUITE_TYPE': JSON.stringify(project),
+                      'process.env.VERSION': JSON.stringify(suiteVersion),
+                      'process.env.COMMITHASH': JSON.stringify(gitRevision),
+                      'process.env.ASSET_PREFIX': JSON.stringify(assetPrefix),
+                      'process.env.JWS_PUBLIC_KEY': JSON.stringify(jwsPublicKey),
+                      'process.env.CODESIGN_BUILD': isCodesignBuild,
+                      'process.env.SENTRY_RELEASE': JSON.stringify(sentryRelease),
+                      __SENTRY_DEBUG__: false,
+                      __SENTRY_TRACING__: false, // needs to be removed when we introduce performance monitoring in trezor-suite
+                  },
+        ),
```

Comment:
I don't like the duplication, what about something like
```suggestion
         new webpack.DefinePlugin({
             'process.browser': true,
             'process.env.SUITE_TYPE': JSON.stringify(project),
             'process.env.VERSION': JSON.stringify(suiteVersion),
             'process.env.COMMITHASH': JSON.stringify(gitRevision),
             'process.env.ASSET_PREFIX': JSON.stringify(assetPrefix),
             'process.env.JWS_PUBLIC_KEY': JSON.stringify(jwsPublicKey),
             'process.env.CODESIGN_BUILD': isCodesignBuild,
             'process.env.SENTRY_RELEASE': JSON.stringify(sentryRelease),
             ...(isDev ? { 
                __SENTRY_DEBUG__: false,
                __SENTRY_TRACING__: false, // needs to be removed when we introduce performance monitoring in
             } : {}
         }),
```

Reply:
How about how it is now?